apps/sms/views/shared/js/activity_handler.js

```diff
@@ -142,7 +144,32 @@ var ActivityHandler = {
       return;
     }
 
-    this.toView({ body: dataToShare });
+    return this.toView({ body: dataToShare });
+  },
+
+  /**
+   * Store a draft for this message, and returns a draftId.
+   *
+   * @param {Object} message to save
+   * @param {(String|Attachment|Array.<Attachment>)} [message.body] The data to
+   * add to the message's body.
+   * @param {String} [message.number] The recipient for this message.
+   * @returns {Promise.<Number>} The new draft id.
+   */
+  _storeDraft(message) {
```

Comment:
I renamed to storeDraftFromMessage

>  I can't think of any cases when it's bad at the moment
Only thing I don't like is that we currently _see_ the draft being created in the inbox before we enter the composer view. But I haven't really looked closely in latest version, maybe now it's fixed because we don't display the inbox before displaying the composer ?